.storybook/recipes/GlobalHeader/GlobalHeader.tsx

```diff
@@ -51,8 +50,44 @@ export interface Props {
   curriculumIsActive?: boolean;
 }
 
+const Avatar = () => (
+  <div className={styles['global-header__avatar-image']}>
```

Comment:
In the avatar components, it was possible to pass in a custom `className`, but we weren't using it.